test/integration/running/test_running.py

```diff
@@ -398,6 +399,44 @@ def test_start_stop_status_cfg(start_stop_cli, project_without_dependencies):
     assert status.get(ID2) == STATUS_STOPPED
 
 
+def test_start_stop_custom_tarantool(start_stop_cli, project_without_dependencies):
+    project = project_without_dependencies
+    cli = start_stop_cli
+
+    INSTANCE1 = 'instance-1'
+    RUN_DIR = 'my-run'
+
+    # Below we are making a bash script named 'tarantool' that calls the real tarantool.
+    # We also change $PATH variable so that the priority of calling our
+    # fake tarantool script is always higher than the original tarantool.
+    # As a result, our process is not called a 'tarantool'.
+
+    real_tarantool_abs_path = find_file_in_path(os.environ['PATH'], 'tarantool')
+    tarantool_substitution_script = f"#!/bin/bash\n{real_tarantool_abs_path} $1"
+    with open('tarantool', 'w') as f:
+        f.write(tarantool_substitution_script)
+
+    os.chmod('tarantool', 0o755)
+
+    old_path = os.environ['PATH']
+    os.environ['PATH'] = os.getcwd() + ':' + old_path
+
+    logs = cli.start(project, [INSTANCE1], stateboard=True, daemonized=True, capture_output=True, run_dir=RUN_DIR)
+
+    os.environ['PATH'] = old_path
+    os.remove('tarantool')
+
+    # Linux, unlike Darwin, which will have a process name 'sh',
```

Comment:
Maybe smth like `On Linux process have name 'tarantool' anyway, but for Darwin it's 'sh', so we can check that warning was displayed`?